include/odp/api/spec/stash_types.h

```diff
@@ -265,6 +265,15 @@ typedef struct odp_stash_param_t {
 	 */
 	odp_stash_stats_opt_t stats;
 
+	/**
+	 * Strict size
+	 *
+	 * If true, application never attempts to store more handles into the stash than specified
+	 * in the 'num_obj' parameter. Depending on the implementation this may enable performance
```

Comment:
Should there be some sort of a reference about `strict_size` in `num_obj` spec, e.g. an "@see"?